Makefile

```diff
@@ -469,8 +469,6 @@ validate_module_configs:
 
 clean:
 	@rm -rf "$(SRC_DIR)"/build
-	@git clean -df -e ".project" -e ".cproject" -e ".idea" -e ".project" -e ".settings" -e ".vscode"
-	@git submodule foreach git clean -dfX
```

Comment:
We need to keep it for the submodules or do something else, things like libuavcan (and others) get into an inconsistent state and it leaves users quite confused.

Reply:
Confused users can use `make distclean` or `make submodulesclean`.